tracer/test/Datadog.Trace.Security.IntegrationTests/AspNetMvc5.cs

```diff
@@ -54,16 +54,15 @@ public AspNetMvc5ClassicWithoutSecurity(IisFixture iisFixture, ITestOutputHelper
     public abstract class AspNetMvc5 : AspNetBase, IClassFixture<IisFixture>
     {
         private readonly IisFixture _iisFixture;
-        private readonly bool _enableSecurity;
         private readonly string _testName;
 
         public AspNetMvc5(IisFixture iisFixture, ITestOutputHelper output, bool classicMode, bool enableSecurity)
             : base(nameof(AspNetMvc5), output, "/home/shutdown", @"test\test-applications\security\aspnet")
         {
             SetSecurity(enableSecurity);
+            _enableSecurity = enableSecurity;
```

Comment:
You deleted that field above didn't you?

Reply:
yes thanks 🤦